manifests/integreatly-3scale/image_mirror_mapping

```diff
@@ -0,0 +1,10 @@
+registry-proxy.engineering.redhat.com/rh-osbs/3scale-mas-3scale-rhel7-operator@sha256:06c81544f81f8d7e4e4c5d44c9ab9f4109f1eadcc24b021ce031a853f0f9e2d3 quay.io/integreatly/delorean:3scale-mas-3scale-rhel7-operator_06c81544f81f8d7e4e4c5d44c9ab9f4109f1eadcc24b021ce031a853f0f9e2d3
```

Comment:
will the mapping still be used?

Reply:
Yep, in order to set up a testing cluster for QE before the images are GA'd we need the mirror mapping still.